dist/templates/literature/results.html

```diff
@@ -53,6 +53,11 @@ <h2 class="record-brief-title">
           </span>
         </div>
       </div>
+      <div ng-if="vm.ClaimToggled" class="col-md-3 right-column">
+        <button ng-if="!vm.alreadyClaimed(record.id) && vm.isClaimable(record.id)" type="button" ng-click="vm.claimRecord(record.id)" class="btn btn-success btn-claim">Claim this record</button>
```

Comment:
Both buttons on click seem to do the same action `vm.claimRecord(record.id)` so what is the difference?